### PR TITLE
Add ARM architecture support for APIM docker images

### PR DESCRIPTION
**Pull request by phiz71**
## Issue

N/A

## Description

 - Use the new graviteeio/nginx base image for UI (see https://github.com/gravitee-io/gravitee-docker/blob/master/images/nginx/Dockerfile)
 - Use `docker buildx build` command to build images for different platforms

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/support-arm-architecture-in-docker-images/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-dlkvpynbvr.chromatic.com)

### CODE REVIEW

**Review by sebdevaux — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -76,6 +76,14 @@ executors:
         resource_class: <<parameters.resource_class>>
 
 commands:
+    create_docker_context:
```
Comment (gaetanmaisse) [inline .circleci/config.yml]: To be consistent with other commands:
```suggestion
    create-docker-context:
```

File: .circleci/Workflows.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+    webUIConsole --> join_state_e2e_cypress
+    webUIPortal --> join_state_e2e_cypress
+    buildImages --> join_state_e2e_cypress
+    e2eCypress: Run Cypress UI tests
+    join_state_e2e_cypress --> e2eCypress
+```
+
+## Merge on master and support branches Workflow
+
+Here is the workflow for a merge on master or on a support brnach.
```
Comment (gaetanmaisse) [inline .circleci/Workflows.md]:
```suggestion
Here is the workflow triggered after a merge on the `master` or a support branch.
```

File: .circleci/Workflows.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+    build --> e2eSDK
+    e2eLintBuild: Lint & build APIM e2e
+    perfLintBuild: Lint & build APIM perf
+    e2eSDK --> e2eLintBuild
+    e2eSDK --> perfLintBuild
+```
+
+## Merge PR and run E2E PR Workflow
+
+Here is the workflow for a merge PR and a "run E2E" PR.
```
Comment (gaetanmaisse) [inline .circleci/Workflows.md]:
```suggestion
Here is the workflow triggered after a push on any branch containing `merge` or `run-e2e` in its name.
```

**Review by gaetanmaisse — APPROVED** [review]
I'm not able to remember why `console-webui-chromatic-deployment` requires:
```yaml
                      - Build Console Storybook
                      - test-repository
                      - test-plugin
                      - test
``` 
So maybe we should keep only `Build Console Storybook`, wdyt?

I just added a couple of small comments. Otherwise, it looks good, CircleCI workflow is 🟢  and we've checked in Azure the images is properly pushed

**Comment by phiz71** [comment]
> console-webui-chromatic-deployment

I think it was because we considered useless to launch chromatic tests if backends tests were failing

**Pull request merged**